CMakeLists.txt

```diff
@@ -372,7 +372,7 @@ install(FILES CHANGELOG.md README.md
 
 include(CMakePackageConfigHelpers)
 write_basic_package_version_file(${CMAKE_BINARY_DIR}/cmake/MaterialXConfigVersion.cmake
-                                 VERSION ${CMAKE_VERSION}
+                                 VERSION ${MATERIALX_LIBRARY_VERSION}
```

Comment:
@JGamache-autodesk, there are some changes to version number setting from main and am preparing a back-merge so let's hold this until this is ready to make sure this would still work. Thanks.